### PR TITLE
[wasm] Fix icall linking with netcore.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: sdks/wasm/src/driver.c [hunk 1]
```diff
@@ -280,8 +279,10 @@ icall_table_lookup (MonoMethod *method, char *classname, char *methodname, char
 	void *p = bsearch (&token_idx, indexes, indexes_size, 4, compare_int);
 	if (!p) {
 		return NULL;
+		/*
```
Comment (kg) [inline sdks/wasm/src/driver.c:282]: is this intentional?

File: sdks/wasm/src/driver.c [hunk 2]
```diff
@@ -259,13 +259,12 @@ icall_table_lookup (MonoMethod *method, char *classname, char *methodname, char
 
 	const char *image_name = mono_image_get_name (mono_class_get_image (mono_method_get_class (method)));
 
-#ifdef ICALL_TABLE_mscorlib
```
Comment (kg) [inline sdks/wasm/src/driver.c:262]: is this intentional?

File: sdks/wasm/src/driver.c (hunk 1 above)
Comment (vargaz) [reply sdks/wasm/src/driver.c:282]: Its unreachable code.

File: sdks/wasm/src/driver.c (hunk 2 above)
Comment (vargaz) [reply sdks/wasm/src/driver.c:262]: Yes, that is always defined.

File: sdks/wasm/src/driver.c (hunk 1 above)
Comment (kg) [reply sdks/wasm/src/driver.c:282]: ok, i just meant 'is it intentional for it to be a comment instead of deleted'

Comment (vargaz) [reply sdks/wasm/src/driver.c:282]: Its debug code which can be enabled.

**Comment by vargaz** [comment]
@monojenkins build failed

**Pull request merged**